app/models/user.rb

```diff
@@ -4,15 +4,16 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
 
-  validates :name, presence: true
-  validates :username, presence: true, uniqueness: true
+  # validates :name, presence: true
+  validates :username, uniqueness: true
   validates :availability, inclusion: { in: [true, false] }
 
   has_many :hangouts
   has_many :hangout_invitees
   has_many :hangouts
   has_many :friend_groups
   has_many :group_members
+  # has_many :friendships
```

Comment:
This does seem correct though maybe we should verify with someone? It would make sense to be able to call friendships on a user instance.

Reply:
Yep, agreed, I got confused whether we could do this with the sender and receiver thing :p